components/list/ListItem.tsx

```diff
@@ -11,6 +11,8 @@ export interface ListItemProps
   onPress?: () => void;
   onPressIn?: () => void;
   onPressOut?: () => void;
+  delayLongPress?: number;
+  onLongPress?: () => void;
```

Comment:
onLongPress?: (event: GestureResponderEvent) => void;

Reply:
I think should amend ListItemProps.

```
onLongPress?: (event: GestureResponderEvent) => void;
onPress?: (event: GestureResponderEvent) => void;
onPressIn?: (event: GestureResponderEvent) => void;
onPressOut?: (event: GestureResponderEvent) => void;
```

Reply:
@bang88

Reply:
@bang88 Updated.